src/fsharp/TypedTree.fs

```diff
@@ -235,6 +231,10 @@ type ValFlags(flags: int64) =
 
     member x.WithIgnoresByrefScope                     =  ValFlags(flags ||| 0b10000000000000000000L)
 
+    member x.InlineIfLambda                            =         (flags &&& 0b100000000000000000000L) <> 0L
```

Comment:
Just to confirm, we do want to include this in the pickled bits? Double checking as it's another flag we have to keep track of for F# metadata.

Reply:
Yes we do, because we need the bit present when we inline something across assembly boundaries whose argument has this